### PR TITLE
[INLONG-4403][Manager] Support Oracle sink

### LINKED ISSUE #4403: [Feature][Manager] support oracle sink
### Description

manager support oracle sink ,add some utils and object to support oracle load node in sort

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by LvJiancheng**
### Prepare a Pull Request
*(Change the title refer to the following example)*

- Title Example: [INLONG-4403] [Manager] Support oracle sink

*(The following *XYZ* should be replaced by the actual [GitHub Issue](https://github.com/apache/incubator-inlong/issues) number)*

- Fixes #4403 

### Motivation

*Support oracle sink*

### Modifications

*Add Iceberg sink info and load node utils.*

### Verifying this change

*(Please pick either of the following options)*

- [ ] This change is a trivial rework/code cleanup without any test coverage.

- [ ] This change is already covered by existing tests, such as:
  *(please describe tests)*

- [ ] This change added tests and can be verified as follows:

  *(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (10MB)*
  - *Extended integration test for recovery after broker failure*

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/oracle/OracleSinkDTO.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    /**
+     * Get Oracle sink info from JSON string
+     */
+    public static OracleSinkDTO getFromJson(@NotNull String extParams) {
+        try {
+            OBJECT_MAPPER.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+            return OBJECT_MAPPER.readValue(extParams, OracleSinkDTO.class);
+        } catch (Exception e) {
+            throw new BusinessException(ErrorCodeEnum.SINK_INFO_INCORRECT.getMessage());
```
Comment (gong) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/oracle/OracleSinkDTO.java:89]: print full exception stack by log

File: inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/oracle/OracleSinkDTO.java [hunk 2]
```diff
@@ -40,8 +43,9 @@
 public class OracleSinkDTO {
 
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    private static final Logger LOGGER = LoggerFactory.getLogger(MySQLSinkDTO.class);
 
-    @ApiModelProperty("Oracle meta db URL, etc jdbc:oracle://host:port/database")
+    @ApiModelProperty("Oracle JDBC URL, such as jdbc:oracle://host:port/database")
```
Comment (gong) [inline inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/oracle/OracleSinkDTO.java:48]: jdbc url example may be error

Comment (LvJiancheng) [reply inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/pojo/sink/oracle/OracleSinkDTO.java]: done

**Review by healchow — APPROVED** [review]
LGTM

**Review by gong — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Review by yunqingmoswu — APPROVED** [review]

**Pull request merged**